Documentation/ColumnsConfig/Type/Input/Properties/Eval.rst

```diff
@@ -175,22 +175,24 @@ saving the record:
       }
    }
 
-:file:`EXT:extension/ext_localconf.php`:
-
 .. code-block:: php
+   :caption: EXT:extension/ext_localconf.php
 
-   // Register the class to be available in 'eval' of TCA
-   $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['tce']['formevals']['Vendor\\Extension\\Evaluation\\ExampleEvaluation'] = '';
+   use Vendor\Extension\Evaluation\ExampleEvaluation;
 
-:file:`EXT:extension/Configuration/TCA/tx_example_record.php`:
+   // ....
+
+   // Register the class to be available in 'eval' of TCA
+   $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['tce']['formevals'][ExampleEvaluation::class] = '';
 
 .. code-block:: php
+   :caption: EXT:extension/Configuration/TCA/tx_example_record.php
 
    'columns' => [
       'example_field' => [
          'config' => [
             'type' => 'text',
```

Comment:
Shall we keep the required option, just to make ppl used to the change?

```suggestion
            'type' => 'text',
            'required' => true,
```

Reply:
Is strictly speaking not part of "eval" but I think that is a good idea.

Reply:
The type is not part of eval either, these are just surrounding information for easier grasp